src/fr/composants/bouton/case-dusage.md

```diff
@@ -17,4 +17,123 @@ permalink: /fr/composants/bouton/
 date: "git Last Modified"
 ---
 
-## Usage
\ No newline at end of file
+Jetez un coup d'œil à ce que font les boutons pour vérifier s'ils permettent de résoudre votre problème. Sélectionnez ensuite le meilleur sous-type, appelé « rôle du bouton », pour le cas d'utilisation dont vous avez besoin.
+
+## Types de problèmes résolus par les boutons
+
+Utilisez un bouton pour les actions importantes qu'une personne utilisant votre produit peut entamer pour :
+
+- donner un ordre lié aux données d'un formulaire;
+- faire un choix ou donner son consentement;
+- entamer un processus, comme un tutoriel, ou se connecter à un espace sécurisé;
+- naviguer entre les différentes étapes d'une séquence;
+- sauter les éléments de navigation pour passer au contenu principal;
+- se rendre vers d'autres pages ou vers des sites externes à l'aide d'un bouton de type « Lien ».
```

Comment:
Can you capitalize the first letter of each bullet point, please?
They were playing around with lower and uppercase letters for bullet points for a while and in the end decided to use an uppercase letter at the beginning of each bullet point but seems like it didn't get updated in all Figma files.